pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -236,6 +236,7 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				// push and pull images
 				authorizationapi.NewRule("get", "update").Groups(imageGroup).Resources("imagestreams/layers").RuleOrDie(),
 				authorizationapi.NewRule("create").Groups(imageGroup).Resources("imagestreamimports").RuleOrDie(),
+				authorizationapi.NewRule("create", "delete").Groups(imageGroup).Resources("imagesignatures").RuleOrDie(),
```

Comment:
this is a cluster level resource, so you shouldn't add it here.  It won't have an effect.